jerry-core/vm/vm.c

```diff
@@ -1578,20 +1578,9 @@ vm_loop (vm_frame_ctx_t *frame_ctx_p) /**< frame context */
           continue;
         }
         case VM_OC_PLUS:
-        {
-          result = opfunc_unary_plus (left_value);
-
-          if (ECMA_IS_VALUE_ERROR (result))
-          {
-            goto error;
-          }
-
-          *stack_top_p++ = result;
-          goto free_left_value;
-        }
         case VM_OC_MINUS:
         {
-          result = opfunc_unary_minus (left_value);
+          result = opfunc_unary_operation (left_value, VM_OC_GROUP_GET_INDEX (opcode_data) - VM_OC_PLUS);
```

Comment:
What if we just had `VM_OC_GROUP_GET_INDEX (opcode_data) == VM_OC_MINUS` or `VM_OC_GROUP_GET_INDEX (opcode_data) != VM_OC_PLUS` here? Would be easier to read, it would impose no ordering on the enum, and the compiler could just as easily optimize it to a subtraction if that's faster on the target architecture.